cli/lib/compiler/Compiler.dart

```diff
@@ -94,8 +97,37 @@ class Compiler {
         .cast<File>();
 
     await for (final file in filesToCopy) {
-      await file.copy(
-          path.join(destinationDir.absolute.path, path.basename(file.path)));
+      if (!(this._config.destinationFilesExclude?.contains(file.path
+              .replaceAll(baseFilesDir.absolute.path, "")
+              .replaceAll("\\", "/")) ??
+          true)) {
+        await file.copy(
+            path.join(destinationDir.absolute.path, path.basename(file.path)));
+      }
+    }
+
+    for (final includePath
+        in this._config.destinationFilesInclude ?? <String>[]) {
+      final file = File(path.join(
+          baseFilesDir.absolute.path,
+          path.relative(
+              includePath.replaceFirst("/", "./").replaceFirst("\\", ".\\"))));
+
+      if (!file.existsSync()) {
+        return;
+      }
+
+      final copyFile = File(path.join(
+          destinationDir.absolute.path,
+          path.relative(
+              includePath.replaceFirst("/", "./").replaceFirst("\\", ".\\"))));
+
+      copyFile.createSync(recursive: true);
+
+      await file.copy(path.join(
+          destinationDir.absolute.path,
+          path.relative(
+              includePath.replaceFirst("/", "./").replaceFirst("\\", ".\\"))));
```

Comment:
`replaceFirst("/", "./").replaceFirst("\\", ".\\")` stuff like that is used very often. Maybe creating util method will be cool?